pkg/volume/flocker/flocker.go

```diff
@@ -421,6 +421,10 @@ func (c *flockerVolumeUnmounter) TearDown() error {
 
 // TearDownAt unmounts the bind mount
 func (c *flockerVolumeUnmounter) TearDownAt(dir string) error {
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
+		glog.Infof("The dir %v doesn't exist as it might have been unmounted by a previous unmount operation", dir)
```

Comment:
If we are going to use `Infof` here - we will be replacing one error message with another. It may be worth making this log only if log levels are higher?